src/resolver.h

```diff
@@ -1 +1,13 @@
+#include "types.h"
+
 char* resolve(char*);
+
+header* build_header();
+
+void reset_counter();
+
+int count_words(char*);
+
+void append(char*, char);
+
+char** split_by_period(char*);
```

Comment:
Why do we export these functions from resolver? Is the plan to have something in the domain outside the resolver use these methods?